chapters/classes.tex

```diff
@@ -344,18 +344,14 @@ \subsection{Component Variability Prefixes discrete, parameter, constant}\label{
 \item
   A variable \lstinline!vc! declared with the \lstinline!parameter!\indexinline{parameter} prefix remains constant during transient analysis, with a value determined by the initialization problem.
 \item
-  A \emph{discrete-time} variable \lstinline!vd! has a vanishing time derivative between events.
-  Note that this is not the same as saying that \lstinline!der(vd)=0! almost everywhere,
-  as the derivative is not even defined at the events, and it is not legal
-  to apply \lstinline!der! to discrete-time variables as they are not continuous. During transient analysis the variable
-  can only change its value at event
-  instants (see \cref{events-and-synchronization}).
+  A \emph{discrete-time} variable \lstinline!vd! is a variable declared with the \lstinline!discrete!\indexinline{discrete} prefix, or a variable without any prefix that is not a continuous-time variable.
+  It has a vanishing time derivative between events.
+  Note that this is not the same as saying that \lstinline!der(vd)=0! almost everywhere, as the derivative is not even defined at the events, and it is not legal to apply \lstinline!der! to discrete-time variables as they are not continuous.
+  During transient analysis the variable can only change its value at event instants (see \cref{events-and-synchronization}).
 \item
-  A \emph{continuous-time} variable \lstinline!vn! may have a non-vanishing time
-  derivative (\lstinline!der(vn)<>0! possible) and may also
-  change its value discontinuously at any time during transient analysis
-  (see \cref{events-and-synchronization}). If there are any discontinuities the variable is
-  not differentiable.
+  A \emph{continuous-time} variable is a \lstinline!Real! variable without any prefix that is not assigned in a \lstinline!when!-clause.
+  A continuous-time variable \lstinline!vn! may have a non-vanishing time derivative (\lstinline!der(vn)<>0! possible) and may also change its value discontinuously at any time during transient analysis (see \cref{events-and-synchronization}).
+  If there are any discontinuities the variable is not differentiable.
```

Comment:
What exactly is this statement about? Mathematical differentiability?  Applicability of `der`?

It would seem more relevant to state something directly related to applicability of `der`, if anything.

(Discussing this as part of this PR only makes sense in view of also trying to get the `der` part right regarding the discrete-time variables.)

Reply:
> What exactly is this statement about? Mathematical differentiability? Applicability of `der`?

Mathematical differentiability. The problem with applicability of `der` is that it is difficult/impossible to **statically** determine if there are any discontinuities or not.

Reply:
OK, but then I think one would need to clarify why we are making statements about mathematical differentiability here, if this doesn't connect with any of the other concepts in the specification, such as `der`.

Reply:
This still remains unresolved.

Reply:
This is now clarified; the point was to state that a continuous-time variable is general without any restrictions on time-evolution, so its trajectory may have parts that continuously change and also parts with discontinuities.

Reply:
The clarification looks good, so then I'm just missing this part:
```suggestion
  Note that \lstinline!der(vn)! is not defined If there are any discontinuities, see \cref{modelica:der}.
```

Reply:
Looks good now.  Resolving.